### PR TITLE
Tablet portrait mode for presets screen

### PR DESCRIPTION
**Pull request by jequeen**
- Adding support for portrait tablet on presets screen
- Moved gridlayout info to integers file

### CODE REVIEW

File: app/src/main/res/drawable/category_group_background.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <shape xmlns:android="http://schemas.android.com/apk/res/android"
     android:shape="rectangle">
-    <corners android:radius="4dp" />
+    <corners android:radius="8dp" />
```
Comment (MattCorbett488) [inline app/src/main/res/drawable/category_group_background.xml]: If you see this radius being used as a standard (I see it as a radius in 2 files in this PR), could be useful to extract it to a dimen so you're not updating every spot every time this changes

**Comment by CarolineLaw** [comment]
so the files named `land` are landscape and everything else is portrait?

**Comment by nicholas-cook** [comment]
@CarolineLaw Yep! If the directory doesn't end with `-land`, the OS will assume portrait

File: app/src/main/res/drawable/category_group_background.xml (hunk 1 above)
Comment (jequeen) [reply app/src/main/res/drawable/category_group_background.xml]: That is a fair point, I pulled those out into the `values/dimens` file. Thanks for the feedback!

**Pull request merged**